data/themes/dark.json

```diff
@@ -14,6 +14,18 @@
     "primary": "#cccccc",
     "secondary": "#a7a8a9"
   },
+  "monitorChart": {
```

Comment:
Nit: Could we keep the lowercase letters since that was the format that already exists? Not that big of a deal but thought it would be better to remain consistent.

Reply:
Good call! I'll address this in #2700